### PR TITLE
MM-59822 - Updating set status button alignment

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-59822 - Updating set status button alignment

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-58182

#### Screenshots
CleanShot 2024-10-14 at 20 48 12@2x

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/profile_popover/profile_popover.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
             background: unset;
             padding-inline-start: 8px;
 
-            > span {
+            >span {
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/profile_popover/profile_popover.scss:30]: can you revert these changes

File: webapp/channels/src/components/profile_popover/profile_popover.scss [hunk 2]
```diff
@@ -103,8 +103,8 @@
             padding-inline-start: 16px;
             text-align: center;
 
-            > h5 {
-                margin:0;
+            >h5 {
+                margin: 0;
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/profile_popover/profile_popover.scss:107]: even this as well

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/profile_popover/profile_popover.scss (hunk 2 above)
Comment (harshilsharma63) [reply webapp/channels/src/components/profile_popover/profile_popover.scss:107]: I believe these arse valid changes via the lint fixer, as there should be a space between attribute and value., and for previous one, there shouldn't be space between `>` and selector, so changes are valid, even if unrelated to the PR.

**Review by harshilsharma63 — COMMENTED** [review]
Tested locally, looks nice!

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
The lint errors can be auto-fixed.

**Review by M-ZubairAhmed — APPROVED** [review]
